### ISSUE TITLE
Open sourcing the design assets 

### ISSUE THREAD

**Issue by bluesign** [onflow/flow#673]
## Instructions

Flow has really nice design language, especially the design of flow-playground and dev-wallet is top of its class in my opinion. It would benefit community to open source this design language ( to people are developing for flip-fest etc )

It will allow those changes to be consistent with the overall design language, and can speed up development for developers that lacking design capabilities.

**Comment by mwufi** [comment]
yeah, i love the flow blockchain explorer! it'd be really nice to open source it + extend it all over the place!
